### ISSUE TITLE
Cellular comparments are not saved/restored

### ISSUE THREAD

**Issue by AlexanderPico** [wikipathways/cytoscape-wikipathways-app#65]
Open WP2884. Note nucleus compartment. Save. Close. Reopen.  Nucleus gone :(

**Comment by AlexanderPico** [comment]
Confirmed. Still a bug.

**Comment by AlexanderPico** [comment]
More details:

1. clone network will lose all annotations
2. save/restore will lose CUSTOM annotations

Adam is also opening a redmine ticket...

**Comment by AlexanderPico** [comment]
This one is fixed in 3.7:
2. save/restore will lose CUSTOM annotations

Cloning is still broken.

**Comment by AdamStuart** [comment]
```
ava.lang.ClassCastException: java.lang.String cannot be cast to org.cytoscape.view.presentation.property.values.Bend
	at org.cytoscape.view.presentation.property.EdgeBendVisualProperty.toSerializableString(EdgeBendVisualProperty.java:44) ~[70:org.cytoscape.api-bundle:3.7.0.SNAPSHOT]
	at org.cytoscape.io.internal.util.vizmap.VisualStyleSerializer.createVizmapProperties(VisualStyleSerializer.java:287) [88:org.cytoscape.io-impl:3.7.0.SNAPSHOT]
	at org.cytoscape.io.internal.util.vizmap.VisualStyleSerializer.createVizmap(VisualStyleSerializer.java:111) [88:org.cytoscape.io-impl:3.7.0.SNAPSHOT]
	at org.cytoscape.io.internal.write.vizmap.VizmapWriterImpl.run(VizmapWriterImpl.java:77) [88:org.cytoscape.io-impl:3.7.0.SNAPSHOT]

```
